src/cosmos/cosmosTypes.ts

```diff
@@ -117,7 +117,7 @@ export interface CosmosOtherMethods {
 export interface CosmosClients {
   queryClient: ReturnType<StargateClient['forceGetQueryClient']>
   stargateClient: StargateClient
-  // Using the tendermint client directly allows us to control the paging
+  // Using the cometClient client directly allows us to control the paging
```

Comment:
I like how there's a typo in the commit message for this one. I appreciate the subtle humor.